src/common/AndroidUtils.ts

```diff
@@ -720,6 +709,26 @@ export class AndroidUtils {
                 return Promise.resolve();
             }
 
+            // Ensure value for runtime.network.latency is in lowercase.
+            // Otherwise the device may not be launchable via AVD Manager.
+            const networkLatency = config.get('runtime.network.latency');
+            if (networkLatency) {
+                config.set(
+                    'runtime.network.latency',
+                    networkLatency.trim().toLocaleLowerCase()
+                );
+            }
+
+            // Ensure value for runtime.network.speed is in lowercase.
+            // Otherwise the device may not be launchable via AVD Manager.
+            const networkSpeed = config.get('runtime.network.speed');
+            if (networkSpeed) {
+                config.set(
+                    'runtime.network.speed',
+                    networkSpeed.trim().toLocaleLowerCase()
+                );
+            }
+
```

Comment:
If we create a new AVD using `AndroidUtils.createNewVirtualDevice()` then users cannot launch the AVD from within AVD Manager. They can only be launched from terminal. After a lot of debugging I figured out the issue:

1. A while ago we switched from using `tools/avdmanager` to `cmdline-tools/avdmanager` since it seems that `tools` are apparently "deprecated" in favor of `cmdline-tools` and are not being maintained.
2. When you use `cmdline-tools/avdmanager` to create a new AVD, it created a `config.ini` file for the AVD that has the following 2 entries in it:
```
runtime.network.latency=None
runtime.network.speed=Full
```
3. The problem with the above is that the values should be in lower case (`none`, `full`) otherwise when the user tries to launch the AVD using AVD Manager, it won't launch and instead errors out (but you can still launch the AVD using the `emulator` command from terminal)
4. So here we are explicitly changing those values to lower case.

Leave it to Google to create tools that break their own other tools 🤦